Trains.NET.Rendering/TrackRenderer.cs

```diff
@@ -8,6 +8,9 @@ internal class TrackRenderer : ITrackRenderer
     {
         private const int PlankWidth = 3;
         private const int NumPlanks = 3;
+        private const int NumCornerPlanks = NumPlanks + 1;
+        private const int CornerEdgeOffsetDegrees = 10;
+        private const int CornerStepDegrees = (90 - 2 * CornerEdgeOffsetDegrees) / (NumCornerPlanks - 1);
```

Comment:
```suggestion
        private const int CornerStepDegrees = 
            // We start the steps in from the edge by the CornerEdgeOffsetDegrees (* 2 because two edges)
            (90 - 2 * CornerEdgeOffsetDegrees) 
            // And now just split up the remainder between planks
            / (NumCornerPlanks - 1);
```